docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -64,6 +64,10 @@ You try to connect to Zeebe from both Desktop Modeler _and_ community-supported
 
 Ensure your computer has access to the (remote) network.
```

Comment:
```suggestion
Ensure your computer has access to the (remote) network.

:::tip
If you run against a Camunda 8 free-trial cluster, ensure it is [not paused](../../concepts/clusters.md#auto-pause).
:::
```

Reply:
OK let's do it. I will combine it with the suggestion https://github.com/camunda/camunda-docs/pull/6063#discussion_r2178223938